src/controllers/connection/newConnection.ts

```diff
@@ -332,6 +332,7 @@ export class NewConnectionController extends HTMLController {
           agent_connection_id: agentConnectionId,
           status: 'pending',
           pin_attempt_count: 0,
+          pin_tries_remaining_count: 0,
```

Comment:
why 2 separate counters?

Reply:
because pin_attempt_count is for counting how many times someone has submitted and sent the wrong pin to you. pin_tries_remaining_count is for seeing how many times you have entered the wrong pin/how many entries you have left - this comes in the problem report from whoever issued you the invite